src/server/game/Spells/SpellEffects.cpp

```diff
@@ -5364,6 +5364,25 @@ void Spell::EffectTransmitted(SpellEffIndex effIndex)
         fz = m_caster->GetPositionZ();
     }
 
+    if (m_spellInfo->Id == 9082) // Investigate the XXX - Mage quest
```

Comment:
what does the comment mean?

Reply:
2 Quests for Mage's ( Both Factions ) . Quest names are 
1.Investigate the Alchemist Shop
2.Investigate the Blue Recluse. 
That's why XXX ( To skip writing both Quests names )